client/overview/survey/index.tsx

```diff
@@ -0,0 +1,187 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import {
+	Card,
+	CardBody,
+	Button,
+	TextareaControl,
+	Icon,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import './style.scss';
+import Emoticons from 'wcpay/overview/survey/emoticons';
+import { HorizontalRule } from '@wordpress/primitives';
```

Comment:
nitpick: move @wordpress components to external deps

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/8309/commits/10d6db8c05752f330a159e40054a89a299693b79 . thx